gradle build asciidoc-to-revealjs-example fails

Build with command
```
gw --stacktrace asciidoctor
```

fails with exception:

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':asciidoctor'.
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:69)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:46)
	at org.gradle.api.internal.tasks.execution.PostExecutionAnalysisTaskExecuter.execute(PostExecutionAnalysisTaskExecuter.java:35)
	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:64)
	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:203)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:185)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:66)
	at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:50)
	at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor.process(DefaultTaskPlanExecutor.java:25)
	at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter.execute(DefaultTaskGraphExecuter.java:110)
	at org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTaskExecutionAction.java:37)
	at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
	at org.gradle.execution.DefaultBuildExecuter.access$000(DefaultBuildExecuter.java:23)
	at org.gradle.execution.DefaultBuildExecuter$1.proceed(DefaultBuildExecuter.java:43)
	at org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildExecutionAction.java:32)
	at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
	at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:30)
	at org.gradle.initialization.DefaultGradleLauncher$4.run(DefaultGradleLauncher.java:154)
	at org.gradle.internal.Factories$1.create(Factories.java:22)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:52)
	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:151)
	at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:32)
	at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:99)
	at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:93)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:62)
	at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:93)
	at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:82)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:94)
	at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:43)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:28)
	at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:75)
	at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:45)
	at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:51)
	at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:28)
	at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:43)
	at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:170)
	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:237)
	at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:210)
	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:35)
	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:206)
	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
	at org.gradle.launcher.Main.doAction(Main.java:33)
	at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
	at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:54)
	at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:35)
	at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
	at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
	at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:127)
	at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:56)
Caused by: org.gradle.api.GradleException: Error running Asciidoctor
	at org.asciidoctor.gradle.AsciidoctorTask.processDocumentsAndResources(AsciidoctorTask.groovy:659)
	at org.asciidoctor.gradle.AsciidoctorTask.this$4$processDocumentsAndResources(AsciidoctorTask.groovy)
	at org.asciidoctor.gradle.AsciidoctorTask.processAsciidocSources(AsciidoctorTask.groovy:592)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.doExecute(AnnotationProcessingTaskFactory.java:227)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:220)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.execute(AnnotationProcessingTaskFactory.java:209)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:585)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:568)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
	... 60 more
Caused by: org.jruby.exceptions.RaiseException: (SystemStackError) stack level too deep
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/block_ulist.html.slim:17)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/block_ulist.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/list.rb:25)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:13)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:12)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)
	at RUBY.render(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:96)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/template.rb:190)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/converter/composite.rb:31)
	at RUBY.convert(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/asciidoctor-1.5.2/lib/asciidoctor/abstract_block.rb:71)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:15)
	at org.jruby.RubyArray.each(org/jruby/RubyArray.java:1613)
	at RUBY.__singleton__(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim:14)
	at org.jruby.RubyBasicObject.instance_eval(org/jruby/RubyBasicObject.java:1533)
	at RUBY.__singleton__(Native Method)
	at RUBY.__tilt_2036(/tmp/asciidoctor-gradle-examples/asciidoc-to-revealjs-example/build/download/templates/templates/slim/section.html.slim)
	at org.jruby.RubyMethod.call(org/jruby/RubyMethod.java:128)
	at RUBY.evaluate(/home/alien/.gradle/caches/modules-2/files-2.1/org.asciidoctor/asciidoctorj/1.5.2/39d33f739ec1c46f6e908a725264eb74b23c9f99/asciidoctorj-1.5.2.jar!/gems/tilt-2.0.1/lib/tilt/template.rb:155)



```

--- a/asciidoc-to-revealjs-example/build.gradle
+++ b/asciidoc-to-revealjs-example/build.gradle
@@ -55,7 +55,7 @@ asciidoctor {
     dependsOn jrubyPrepareGems
 
 	sources {
-		include 'example-manual.adoc'
+		include 'example-deck.adoc'
 	}
 
 	resources {

--- a/asciidoc-to-revealjs-example/build.gradle
+++ b/asciidoc-to-revealjs-example/build.gradle
@@ -72,18 +72,18 @@ asciidoctor {
 	attributes	'build-gradle': file('build.gradle'),
 				'sourcedir': project.sourceSets.main.java.srcDirs[0],
 				'endpoint-url': 'http://example.org',
-				'source-highlighter' : 'coderay',
-				'imagesdir':'./images',
-				'toc':'left',
+				'source-highlighter': 'coderay',
+				'imagesdir': './images',
+				'toc': 'left',
 				'icons': 'font',
-				'setanchors':'true',
-				'idprefix':'',
-				'idseparator':'-',
-				'docinfo1':'true',
-				'revealjs_theme':'black',
-				'revealjs_transition':'linear',
-				'revealjs_history':'true',
-				'revealjs_slideNumber':'true'
+				'setanchors': '',
+				'idprefix': '',
+				'idseparator': '-',
+				'docinfo1': '',
+				'revealjs_theme': 'black',
+				'revealjs_transition': 'linear',
+				'revealjs_history': 'true',
+				'revealjs_slideNumber': 'true'
 
 	options template_dirs : [new File(templateDir,'templates/slim').absolutePath ]
 

--- a/asciidoc-to-revealjs-example/build.gradle
+++ b/asciidoc-to-revealjs-example/build.gradle
@@ -6,7 +6,7 @@ buildscript {
 }
 
 plugins {
-  id 'org.asciidoctor.convert' version '1.5.2'
+  id 'org.asciidoctor.convert' version '1.5.3'
   id 'com.github.jruby-gradle.base' version '0.1.5'
 }
 
@@ -30,8 +30,8 @@ repositories {
 }
 
 dependencies {
-    gems 'rubygems:slim:2.1.0'
-    gems 'rubygems:thread_safe:0.3.4'
+    gems 'rubygems:slim:3.0.6'
+    gems 'rubygems:thread_safe:0.3.5'
 }
 
 task download << {

--- a/asciidoc-to-revealjs-example/src/docs/asciidoc/example-deck.adoc
+++ b/asciidoc-to-revealjs-example/src/docs/asciidoc/example-deck.adoc
@@ -1,4 +1,4 @@
-= Example Manual
+= Example Deck
 Doc Writer <doc.writer@example.org>
 2014-09-09
 :revnumber: {project-version}
@@ -9,8 +9,8 @@ ifndef::sourcedir[:sourcedir: ../java]
 == Introduction
 
 [%step]
-* This is a user manual for an example project.
-* This project does something.
+* This is an example deck.
+* This deck teaches something.
 * We just haven't decided what that is yet.
 
 == Speaker Notes
@@ -80,4 +80,3 @@ endpoint-url:: {endpoint-url}
 ----
 include::{build-gradle}[]
 ----
-

